composer.json

```diff
@@ -16,6 +16,7 @@
         "equip/structure": "^1.0",
         "filp/whoops": "^2.0",
         "lukasoppermann/http-status": "^2",
+        "monolog/monolog": "^1.19",
```

Comment:
My concern is that if we don't provide a default configuration that includes an implementation of PSR-3, things will blow up when trying to handle exceptions out of the box.